src/components/MAPI/organizations/OrganizationListCreateOrg.tsx

```diff
@@ -80,7 +82,7 @@ const OrganizationListCreateOrg: React.FC<IOrganizationListCreateOrgProps> = ({
   const handleSubmit = async (e: React.FormEvent<HTMLElement>) => {
     e.preventDefault();
 
-    if (!isValid) return;
+    if (!isValid || !canCreateOrganizations) return;
```

Comment:
Extra check here for the function handling organization creation. Since we are fetching permissions every minute or so, there could be a possibility that from the time the page is rendered to the submission request is made that the user's permissions has changed.